CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed all Google AMP template files [#2308](https://github.com/bigcommerce/cornerstone/pull/2308)
 - Webpack 5, Node 18 Support [#2311][https://github.com/bigcommerce/cornerstone/pull/2311]
 - Customer order summary with both physical and digital items shows shipping as null [#2309](https://github.com/bigcommerce/cornerstone/pull/2309)
+- Fix sold-out badge appearance [#2315](https://github.com/bigcommerce/cornerstone/pull/2315)
```

Comment:
this changelog looks outdated as we already have 6.8.0 release - will it be updated somehow before merge?

Reply:
Yes, it's been updated yesterday after PR was ready to test so I will update before merging